src/services/transitGatewayRouteTable/format.ts

```diff
@@ -0,0 +1,51 @@
+import cuid from 'cuid'
+import { formatTagsFromMap } from '../../utils/format'
+import { RawAwsTransitGatewayRouteTable } from './data'
+import { AwsTransitGatewayRouteTable } from '../../types/generated'
+import { transitGatewayRouteTableArn } from '../../utils/generateArns'
+
+/**
+ * Transit Gateway Route Table
+ */
+
+export default ({
+  service: rawData,
+  account,
+  region,
+}: {
+  service: RawAwsTransitGatewayRouteTable
+  account: string
+  region: string
+}): AwsTransitGatewayRouteTable => {
+  const {
+    TransitGatewayRouteTableId: id,
+    State: state,
+    DefaultAssociationRouteTable: defaultAssociationRouteTable,
+    DefaultPropagationRouteTable: defaultPropagationRouteTable,
+    CreationTime: creationTime,
+    Tags: tags,
+    Routes: routes = [],
+  } = rawData
+
+  const transitGatewayRouteTable = {
+    id,
+    accountId: account,
+    arn: transitGatewayRouteTableArn({ region, account, id }),
+    region,
```

Comment:
Lets add the `transitGatewayId` as part of the data here. It will be helpful in trying to match the services together in code on EP.